pkg/nb/rpc_benchmark_ws.go

```diff
@@ -0,0 +1,53 @@
+package nb
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/noobaa/noobaa-operator/v2/pkg/util"
+)
+
+var client = flag.Bool("client", false, "client option")
+var port = flag.String("port", "5656", "port option")
+var hostname = flag.String("hostname", "127.0.0.1", "hostname option")
+var wsize = flag.Int64("wsize", 1024*1024, "wsize option")
+var rsize = flag.Int64("rsize", 1024*1024, "rsize option")
+var concur = flag.Int("concur", 16, "concur option")
+
+func benchmark() {
```

Comment:
This shouldn't be in the nb package because it would pollute the global flags.
Move this to its own package main under `test/rpcbench/`
Also add a comment on the function how to use it.